in_toto/keylib.go

```diff
@@ -497,14 +525,17 @@ func VerifySignature(key Key, sig Signature, unverified []byte) error {
 		}
 		switch key.Scheme {
 		case rsassapsssha256Scheme:
-			hashed := sha256.Sum256(unverified)
+			if _, ok := hashMapping["sha256"]; !ok {
+				return ErrUnsupportedHashAlgorithm
+			}
+			hashed := hashToHex(hashMapping["sha256"](), unverified)
```

Comment:
No that `hashed` is no longer an array you don't have to convert it to a slice below in `rsa.VerifyPSS` and `ecdsa.Verify`, i.e. you can do `s/hashed[:]/hashed`.